src/schedule.jl

```diff
@@ -305,18 +313,24 @@ function _get_today_tables(;
 end
 
 # A dispatcher for the `today` methods. Defaults to terminal output.
+"""     today(; speaker)
+
+Prints the schedule of today.
+`speaker` can be a string identifying the speaker to filter the schedule.
+"""
 function today(;
     now=default_now(),
+    speaker=nothing,
```

Comment:
I'd much rather see this entire functionality in a separate function `JuliaCon.talks_by(speaker)` rather than as an extra keyword argument.

Also, only showing the talks of a certain speaker on a given day is likely to lead to empty / very sparse output. Typically, people are giving 1-2 talks at all. So filtering the entire JuliaCon schedule seems better IMO.

Reply:
Agree on the sparsity, I'll try to adapt it.